libraries/provider_database_mysql.rb

```diff
@@ -136,22 +120,27 @@ def close_repair_client
         end
 
         def query_client
+          @query_client ||= create_mysql_query_client
+        end
+
+        def close_query_client
+          @query_client.close
+        rescue Mysql2::Error
+          @query_client = nil
+        end
+
+        def create_mysql_query_client
           require 'mysql2'
-          @query_client ||=
-            Mysql2::Client.new(
+          Mysql2::Client.default_query_options[:connect_flags] |= Mysql2::Client::MULTI_STATEMENTS
```

Comment:
This changes the connection flags for *all* connections.
See my comments to 5928ecc0df11d7c50